zcash_primitives/CHANGELOG.md

```diff
@@ -15,6 +15,13 @@ workspace.
 - `zcash_primitives::transaction::tests::data` has been exposed under the
   `test-dependencies` feature to provide access to test vectors.
 
+## [0.24.1] - 2025-09-09
+
+### Changed
+- This release makes no public API changes, but provides pre-release support
```

Comment:
This is incorrect. The exposing of  `zcash_primitives::transaction::tests::data` is released in this point release: